flatpak/manifest_tool/tests/flutter/test_offline_fixes.py

```
@@ -1,15 +1,14 @@
-import pytest
 from pathlib import Path
 import sys
-import tempfile
 
 # Add parent directory to path for imports
 sys.path.insert(0, str(Path(__file__).parent.parent.parent))
 
-from manifest_tool.core.manifest import ManifestDocument
-from manifest_tool.flutter import offline_fixes
+# Import after sys.path modification
+from manifest_tool.core.manifest import ManifestDocument  # noqa: E402
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)